Formula/d/deno.rb

```diff
@@ -43,18 +43,24 @@ class Deno < Formula
   # We use the crate as GitHub tarball lacks submodules and this allows us to avoid git overhead.
   # TODO: Remove this and `v8` resource when https://github.com/denoland/rusty_v8/issues/1065 is resolved
   resource "rusty-v8" do
-    url "https://static.crates.io/crates/v8/v8-0.74.1.crate"
-    sha256 "1202e0bd078112bf8d521491560645e1fd6955c4afd975c75b05596a7e7e4eea"
+    url "https://static.crates.io/crates/v8/v8-0.74.3.crate"
+    sha256 "2eedac634b8dd39b889c5b62349cbc55913780226239166435c5cf66771792ea"
   end
 
   # Use the latest tag in https://github.com/denoland/v8/tags.
   resource "v8" do
-    url "https://github.com/denoland/v8/archive/refs/tags/11.7.439.2-denoland-9fcf117b2e045094627c.tar.gz"
-    sha256 "20661cff2d5f5fe7393d992225f7098bede350bc9415068a811ef08e95205b1b"
+    url "https://github.com/denoland/v8/archive/refs/tags/11.7.439.6-denoland-40ca3145809218c5545c.tar.gz"
+    sha256 "eb0f6a843326b79dd0890b4aadab1c3ecde4a2e097d75964a95e19b498e3c38b"
+  end
+
+  # Use the latest tag in https://github.com/denoland/deno_core/tags.
```

Comment:
Note that this will work most of the time, but is not a guarantee. Ideally the version of deno_core you build from should come from the Cargo.lock file.

Reply:
Thanks for the tip! Yea, let me follow up with that after this PR gets merged (otherwise we'll need to rerun CI).

Reply:
Actually, let me do it here. CI took unusually long on some platforms so we'll need a rerun anyway.